public/vercel/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
The css and half of the HTML is spec to Netlify and should be removed (it was done for demo, cf: https://algoliasearch-netlify.netlify.app/)

Reply:
I'll suggest to clean public/netlify too, I did not paid attention the first time 👌🏻